### ISSUE TITLE
Broken link on breadcrumb bar on Help, About, and Policy pages

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#299]
From the About, Policies, and Help pages, the link to "VTechWorks Home" on the breadcrumb bar is http://vtechworks.lib.vt.edu/page/'/', which leads to the error page. It should be http://vtechworks.lib.vt.edu. These pages all have "page" in their path, e.g., http://vtechworks.lib.vt.edu/page/policies. 

A related issue is that it might be better if we could omit "page" from the path, e.g., http://vtechworks.lib.vt.edu/policies.

**Comment by keithgee** [comment]
@AmberPoo1 If these were working previously, it's possible that I broke them with the commit at https://github.com/VTUL/vtechworks/commit/0219883478f5f2b77a6146f10f5a21807bea569b. I remember rushing slightly to get this in as it was marked as high priority, and I see some odd code with '/' in it. That matches part of the link that @alawvt reported above, and also in some of the places I'm not sure why I've quoted the path.
